internal/controller/controllers/csr_utils.go

```diff
@@ -0,0 +1,80 @@
+package controllers
```

Comment:
How much of this code can we share/reuse from the cluster-machine-approver? 

https://github.com/openshift/cluster-machine-approver/tree/master/pkg

It looks to me that we may be able to vendor CMA and re-use its utilities